updating/updating-cluster-between-minor.adoc

```diff
@@ -38,3 +38,7 @@ include::modules/update-service-overview.adoc[leveloffset=+1]
 include::modules/understanding-upgrade-channels.adoc[leveloffset=+1]
 
 include::modules/update-upgrading-web.adoc[leveloffset=+1]
+
+.Additional resources
+
+* xref::../support/troubleshooting/troubleshooting-upgrades.adoc[Troubleshooting upgrades]
```

Comment:
@ehvs Consider adding cross-references like this one to each the `updating/updating*.adoc` topics. These should bring users from the updating topics to the troubleshooting topics if they run into trouble.